pandas/tests/io/test_fsspec.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import pytest
+
+from pandas import DataFrame, date_range, read_csv, read_parquet
+import pandas._testing as tm
+from pandas.util import _test_decorators as td
+
+df1 = DataFrame(
+    {
+        "int": [1, 3],
+        "float": [2.0, np.nan],
+        "str": ["t", "s"],
+        "dt": date_range("2018-06-18", periods=2),
+    }
+)
+text = df1.to_csv(index=False).encode()
+
+
+@pytest.fixture
+def cleared_fs():
+    import fsspec
+
+    memfs = fsspec.filesystem("memory")
+    try:
+        yield memfs
+    finally:
+        memfs.store.clear()
+
+
+@td.skip_if_no("fsspec")
```

Comment:
Can you `pytest.importorskip` inside the `cleared_fs` fixture? Then you don't need to repeat this.